src/sidebar/components/ShareDialog/ShareDialog.tsx

```diff
@@ -9,94 +10,94 @@ import ShareAnnotations from './ShareAnnotations';
 import TabHeader from './TabHeader';
 import TabPanel from './TabPanel';
 
+// The value for every item here represents the order in which tabs should be
+// rendered.
+// For example, if `share` has the lower value, it will be rendered as the first
+// tab when provided, even if it is provided last in the tabs prop.
+const tabsOrder = {
+  share: 0,
+  export: 1,
+  import: 2,
+} as const;
+
+export type ShareDialogTab = keyof typeof tabsOrder;
+
+export type ShareDialogProps = {
+  tabs: NonEmptyArray<ShareDialogTab>;
```

Comment:
What is the reason to require `tabs` to be non-empty here? The component could simply render nothing if the `tabs` array is empty.